src/serial/parse_input.py

```diff
@@ -1,6 +1,35 @@
-def parse_string(message: str) -> str:
+import json
+
+ACC_GYR_Value = {'x': -1, 'y': -1, 'z': -1}
```

Comment:
should probably move this inside the function because parse_string should actually return a dict and not a string